### ISSUE TITLE
`<link rel="sitemap">`

### ISSUE THREAD

**Issue by ArmorDarks** [h5bp/html5-boilerplate#1895]
Official HTML5 Boilerplate docs [recommends](https://github.com/h5bp/html5-boilerplate/blob/71b062d38a6cc6f60d432d8b32cb29f8c13a76da/dist/doc/extend.md#direct-search-spiders-to-your-sitemap) to use

``` html
<link rel="sitemap" type="application/xml" title="Sitemap" href="/sitemap.xml">
```

for feeding sitemaps to search crawlers.

We've tested such link on few sites, and seems that crawlers do not pick up `sitemap.xml` based on it.

For instance, Google [doesn't list](https://support.google.com/webmasters/answer/183668?hl=en) sitemap link as possible way to refer `sitemap.xml`.

Instead, it recommends to add reference to `robots.txt`:

```
Sitemap: http://example.com/sitemap_location.xml
```

Can someone confirm that sitemap `<link rel="sitemap">` actually works?

**Comment by ArmorDarks** [comment]
We experimented with our projects and put `sitemap.xml` reference into `robots.txt` as Google suggested.

However, even after months it didn't appear in Google Search Console, so it's really hard to tell did it have any effect or no.

Worth mentioning that Google referencing `sitemap.xml` in `robots.txt` too: https://www.google.com.ua/robots.txt

**Comment by roblarsen** [comment]
Closed, yo, via #1918
